docs/reference/module-overview.md

```diff
@@ -0,0 +1,284 @@
+
+### `fields`
+
+[Doc type](/reference/glossary.md#doc) modules have some fields configured by default, such as the `title` and `slug` fields. The `fields` setting is used for additional field management.
+
+The `fields` object is configured with subsections: `add`, `remove`, and `group`.
+
+#### `add`
+
+An object of fields to add to the doc type's schema. See the [field type reference](/reference/field-types/README.md) for more on field type configuration.
+
+```javascript
+// modules/article/index.js
+modules.export = {
+  fields: {
+    add: {
+      subtitle: {
+        label: 'Subtitle',
+        type: 'string'
+      }
+    }
+  }
+};
+```
+
+#### `remove`
+
+An array of field names from the base class module to remove. Some core doc type fields cannot be removed since they required by core functionality.
+
+```javascript
+// modules/spotlight-article/index.js
+modules.export = {
+  extend: 'article',
+  fields: {
+    remove: [ 'subtitle' ]
+  }
+};
+```
+
+#### `group`
+
+An object of field groups. The groupings are used by editing interface. It _does not apply to widget modules_, which do not have a tabbed interface.
+
+Groups are added as an object with their name as the object key and the following properties:
+- `label`: The visible label (a string) on the interface tab
+- `fields`: An array of field names to include in the group
+
+The `@apostrophecms/doc-type` module groups the default fields in a `basics` group, which you can override. If overriding the `basics` group, you will need to add those core fields to a group in the module. Any fields not added to a group will be placed in an "Ungrouped" section in the editing interface.
+
+```javascript
+// modules/article/index.js
+modules.export = {
+  fields: {
+    add: {
+      // ...
+    },
+    group: {
+      meta: { // 👈 The group's identifying name is the object key.
+        label: 'Article metadata',
+        fields: [
+          'subtitle',
+          'author',
+          '_category'
+        ]
+      }
+    }
+  }
+};
+```
+
+### `filters`
+
+In piece type modules, the `filters` setting configures the pieces manager modal, adding and removing filtering interface (to view only certain pieces). `trash` and `visibility` filters are included by default.
+
+The `filters` object is configured with subsections: `add` and `remove`. Filters must correspond to an existing fields name or custom [query builder](#queries-self) on the piece type.
+
+#### `add`
+
+An object of filters to add to the piece type. Each filter is an object with its own configuration. Filter properties include:
+
+| Property | Description |
+| ------- | ------- |
+| `label` | Recommended, but not required, to customize the human-readable label. |
+| `inputType` | Choose an input field type for the filter from the following: `radio`, `checkbox`, or `select`. Defaults to `select`. |
```

Comment:
"Choose an input field type for the filter from the following" -> "Declare the input field-type for the filter. `radio`, `checkbox`, or `select` are valid.  Defaults to `select`"